src/ruby/devcontainer-feature.json

```diff
@@ -21,7 +21,6 @@
     "customizations": {
         "vscode": {
             "extensions": [
-                "rebornix.Ruby",
```

Comment:
This makes sense to me 👍 

Looping in @bamurtaugh for product insights.

Reply:
Makes sense to me too, thank you!